example/debug/odp_debug.c

```diff
@@ -285,49 +345,67 @@ int main(int argc, char *argv[])
 
 	if (argc < 2) {
 		/* If not arguments, run all test cases */
-		global->shm_all = 1;
+		global->system  = 1;
 		global->shm     = 1;
 		global->pool    = 1;
 		global->queue   = 1;
+		global->pktio   = 1;
+		global->ipsec   = 1;
 	} else {
 		if (parse_options(argc, argv, global))
 			return -1;
 	}
 
 	if (odp_init_global(&inst, NULL, NULL)) {
-		printf("Global init failed.\n");
+		ODPH_ERR("Global init failed.\n");
 		return -1;
```

Comment:
While your are cleaning the error messages, you could also use standard return codes. E.g. `exit(EXIT_FAILURE)`.

Reply:
OK. I'll do that as well.